doc/configcommands.dsv

```diff
@@ -8,7 +8,7 @@ bind-key||<key> <operation> [<dialog>]||n/a||Bind key <key> to <operation>. This
 bookmark-autopilot||[yes/no]||no||If set to `yes`, the configured bookmark command is executed without any further input asked from user, unless the url or the title cannot be found/guessed.||bookmark-autopilot yes
 bookmark-cmd||<command>||""||If set, then <command> will be used as bookmarking plugin. See the documentation on bookmarking for further information.||bookmark-cmd "~/bin/delicious-bookmark.sh"
 bookmark-interactive||[yes/no]||no||If set to `yes`, then the configured bookmark command is an interactive program.||bookmark-interactive yes
-browser||<command>||%BROWSER, otherwise lynx||Set the browser command to use when opening an article in the browser. If the <<BROWSER,`BROWSER`>> environment variable is set, it will be used as the default browser, otherwise lynx will be used. Any occurrences of `%u` in <command> will be replaced by a URL in single quotes.||browser "w3m %u"
+browser||<command>||%BROWSER, otherwise lynx||Set the browser command to use when opening an article in the browser. If the <<BROWSER,`BROWSER`>> environment variable is set, it will be used as the default browser, otherwise lynx will be used. Any occurrences of `%u` in <command> will be replaced by the relevant URL in single quotes. Any occurrences of `%F` in <command> will be replaced by the feed's rss URL in single quotes.||browser "w3m %u"
```

Comment:
Two notes on this change:

1. "the relevant URL" feels pretty vague to me, because it's unclear what dictates relevance. I'd use "the URL being opened" or something like this — it doesn't explain which URL will be opened from, say, a feedlist, but it hopefully gives the user enough understanding that they can correctly *guess* what's going to be opened. Does that make sense?

2. I'd drop "rss" from "the feed's rss URL". For one, it looks like a leftover from editing, otherwise you'd probably have "RSS" properly capitalized :) But also because it doesn't add any detail and actually introduces ambiguity ("does this apply to Atom feeds too?").

Reply:
Good points, amended into: https://github.com/newsboat/newsboat/pull/1314/commits/66d984e8aeabb8bf7dd18b6eccc57afaa7bbeda8